content/applications/finance/accounting/taxes/avatax/avalara_portal.rst

```diff
@@ -0,0 +1,255 @@
+the Avalara console, navigate to :menuselection:`Transactions --> Transactions`.
+
+.. image:: avalara_portal/avalara_transactions.png
+   :align: center
+   :alt: Avalara portal with the transactions shortcut highlighted.
+
+Edit transaction
+----------------
+
+Click into a transaction to reveal more details about the transaction. These include sections on
+:guilabel:`Invoice detail`, :guilabel:`Additional info`, and :guilabel:`Customer info`. Click
+:icon:`fa-pencil` :guilabel:`Edit document details` to make changes to the transaction.
+
+A :guilabel:`Discount` can be added to adjust the invoice. This is especially useful in cases where
+the transaction has already synced with Avalara / *AvaTax* and changes need to be made afterward.
+
+.. _avalara_portal/filter:
+
+Filter
+------
+
+Filter transactions by setting the :guilabel:`From` and :guilabel:`To` fields, and configuring other
+fields to filter by, including:
+
+ - :guilabel:`Document Status` : either of the following options, :guilabel:`All`,
+   :guilabel:`Voided`, :guilabel:`Committed`, :guilabel:`Uncommitted`, or :guilabel:`Locked`.
+ - :guilabel:`Document Code` : either of the following options, :guilabel:`Exactly match`,
+   :guilabel:`Starts with`, or :guilabel:`Contains`.
+ - :guilabel:`Customer/Vendor Code`: this is the customer/vendor code in Odoo (e.g. Contact18).
+ - :guilabel:`Country` : this is the country this tax was calculated in; this is a text field.
+ - :guilabel:`Region` : this is the region of the country, this will vary based on the
+   :guilabel:`Country` selection.
+ - :guilabel:`Document Type` : either of the following selections, :guilabel:`All`, :guilabel:`Sales
+   Invoice`, :guilabel:`Purchase Invoice`, :guilabel:`Return Invoice`, :guilabel:`Inventory Transfer
+   Inbound Invoice`, :guilabel:`Inventory Transfer Outbound Invoice`, or :guilabel:`Customs
+   Invoice`.
+ - :guilabel:`Import ID` : represents the import ID of the document.
+
+Sort by
+-------
+
+On the *Transactions* page, transactions will be listed below according to the set
+:ref:`avalara_portal/filter`, in the top half of the page. The following columns are available by
+default, to sort by ascending or descending order:
+
+ - :guilabel:`Doc Code` : either of the following options, :guilabel:`Exactly match`,
+   :guilabel:`Starts with`, or :guilabel:`Contains`.
+ - :guilabel:`Doc Status` : either of the following options, :guilabel:`All`, :guilabel:`Voided`,
+   :guilabel:`Committed`, :guilabel:`Uncommitted`, or :guilabel:`Locked`.
+ - :guilabel:`Cust/Vendor Code` : this is the customer/vendor code in Odoo (e.g. Contact18).
+ - :guilabel:`Region` : this is the region of the country, this will vary based on the
+   :guilabel:`Country` selection.
+ - :guilabel:`Amount` : the numeric amount of the total amount on the Odoo document.
+ - :guilabel:`Tax` : the numeric amount of the tax applied to the total.
```

Comment:
```suggestion
 - :guilabel:`Doc Code`: any of the following options, :guilabel:`Exactly match`,
   :guilabel:`Starts with`, or :guilabel:`Contains`.
 - :guilabel:`Doc Status`: any of the following options, :guilabel:`All`, :guilabel:`Voided`,
   :guilabel:`Committed`, :guilabel:`Uncommitted`, or :guilabel:`Locked`.
 - :guilabel:`Cust/Vendor Code`: the customer/vendor code in Odoo (e.g. `Contact18`).
 - :guilabel:`Region`: the region of the country, which varies based on the
   :guilabel:`Country` selection.
 - :guilabel:`Amount`: the numeric amount of the total amount on the Odoo document.
 - :guilabel:`Tax`: the numeric amount of the tax applied to the total.
```
just another friendly reminder to double-check the accuracy of these sort by columns, as I'm not sure I am seeing all of them on the sandbox you provided the credentials for... just a suggestion

Reply:
It saved some settings from when I accessed the site before, the documented sort by were default.